src/zentables/zentables.py

```diff
@@ -827,14 +893,17 @@ def _combine_n_pct(
     df_n: pd.DataFrame,
     df_pct: pd.DataFrame,
     digits: int = 1,
-    suppress_symbol: str = "*",
+    suppress_symbol: Optional[str] = None,
```

Comment:
Hey Paul. Sorry for the delay here, but is there a reason why the `"*"` is the default value for this variable in the function above and `None` is the default value here?